Week1/homework/app.js

```diff
@@ -1,11 +1,206 @@
+        language: '	English',
+        author: 'Peter Lubber,‎ Frank Salim,‎ Brian Albers',
+    },
+    a_christmas_carol: {
+        title: 'A Christmas Carol',
+        language: '	English',
+        author: 'Charles Dickens',
+    },
+    divergent: {
+        title: 'Divergent',
+        language: '	English',
+        author: 'Veronica Roth',
+    },
+    the_handmaid_tale: {
+        title: 'The Handmaid Tale',
+        language: '	English',
+        author: 'Margaret Atwood',
+    },
+
+}
+//Book List
+const title = document.getElementById('books');
+const h1 = document.createElement('h1');
+h1.innerHTML = 'Book List';
+h1.classList.add("book-header");
+title.appendChild(h1);
+h1.addEventListener("mouseover", () => {
+  h1.style.backgroundColor = "red";
+  h1.style.color = "black";
+});
+h1.addEventListener("mouseout", () => {
+  h1.style.color = "blue";
+  h1.style.backgroundColor = "white";
+});
+
+//1.5 Now change the function from step 1.3 that you used to display the book ID's in a list to take the actual information about the book from the object and display that.
+
+/*function createUlElement(ul) {
+    for (var key in infoOfBooks) {
+        const titleText = infoOfBooks[key].title;
+        const languageText = infoOfBooks[key].language;
+        const authorText = infoOfBooks[key].author;
+        const titleElement = document.createElement('h1');
+        const languageElement = document.createElement('p');
+        const authorElement = document.createElement('p');
+        const li = document.createElement('li');
+        titleElement.innerText = titleText;
+        languageElement.innerText = languageText;
+        authorElement.innerText = authorText;
+        li.appendChild(titleElement);
+        li.appendChild(languageElement);
+        li.appendChild(authorElement);
+        ul.appendChild(li);
+    }
+}
+
+
+function booksId() {
+    const books = document.getElementById('books');
+    const ul = document.createElement('ul');
+    books.appendChild(ul);
+    createUlElement(ul);
+}
+window.addEventListener('load', booksId);*/
+
+
+//1.7 Find and download book covers for each book and construct a new object which has as keys the book IDs again,
+// and as value the path to the image source
+
+const images = {
+    romeo_and_juliet: './img/romeo-juliet.jpg',
+    harry_potter_and_the_sorcerer_stone: './img/harry potter.jpg',
+    the_book_of_css3: './img/css3.jpg',
+    catching_fire: './img/catching_fire_book_cover.jpg',
+    eloquent_javaScript: './img/javascript.jpg',
+    hamlet: './img/hamlet.jpg',
+    pro_html5_programming: './img/html5.jpg',
+    a_christmas_carol: './img/A Christmas Carol.jpg',
+    divergent: './img/divergent.jpg',
+    the_handmaid_tale: './img/The Handmaid Tale.jpg',
+
+}
+
+//1.8 Loop over these entries (hint: Object.keys(objectName) gives you an array containing the keys). 
+//Then write a function which places an image at the corresponding li element
+
+
+function createUlElement(ul) {
```

Comment:
try to find a better name, think less like a programmer in naming things. createBooksList() maybe?